sci-libs/openlibm/openlibm-0.7.5.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="High quality system independent, portable, open source libm implementation"
+DESCRIPTION="OpenLibm is an effort to have a high quality, portable, \
+standalone C mathematical library (libm). It can be used standalone in \
+applications and programming language implementations.
+
+The project was born out of a need to have a good libm for the Julia \
+programming langage that worked consistently across compilers and operating \
+systems, and in 32-bit and 64-bit environments."
+HOMEPAGE="https://openlibm.org"
+COPYRIGHT="2011-14 The Julia Project. \
+2008 Stephen L. Moshier \
```

Comment:
should be indented with a tab

Reply:
You mean the copyright holders on each separate line?

Reply:
yep. But only the extra lines.